tests/publiccloud/azure_cli.pm

```diff
@@ -25,8 +25,11 @@ sub run {
 
     # If 'az' is preinstalled, we test that version
     if (script_run("which az") != 0) {
-        add_suseconnect_product(get_addon_fullname('pcm'), (is_sle('=12-sp5') ? '12' : undef));
-        add_suseconnect_product(get_addon_fullname('phub')) if is_sle('=12-sp5');
+        # Public Cloud module is not needed since SLE 16 to install azure cli
+        unless (is_sle('16+')) {
+            add_suseconnect_product(get_addon_fullname('pcm'), (is_sle('=12-sp5') ? '12' : undef));
```

Comment:
Nitpick: Why not just add the `unless` on this line? The whole block appears unnecessary to me.

This is a style suggestion, so feel free to ignore.